src/main/java/com/github/alex1304/jdash/client/AbstractGDClient.java

```diff
@@ -53,10 +53,7 @@
 		this.host = host;
 		this.client = HttpClient.create()
 				.baseUrl(host)
-				.headers(h -> {
-					h.add("Content-Type", "application/x-www-form-urlencoded");
-					h.add("User-Agent", "");
```

Comment:
Do not remove this line, it was intentionally added to prevent the server from blocking the requests (see 3.3.8 release patch notes)

Reply:
Reverted 5888910